module/Database/src/Model/Member.php

```diff
@@ -749,6 +749,27 @@ public function toArray(): array
         ];
     }
 
+    /**
+     * Get array of member for use in API endpoints
+     *
+     * @return array
+     */
+    public function toArrayApi(): array
+    {
+        return [
+            'lidnr' => $this->getLidnr(),
+            'email' => $this->getEmail(),
+            'fullName' => $this->getFullName(),
+            'lastName' => $this->getLastName(),
+            'middleName' => $this->getMiddleName(),
+            'initials' => $this->getInitials(),
+            'firstName' => $this->getFirstName(),
+            'generation' => $this->getGeneration(),
+            'expiration' => $this->getExpiration()->format('l j F Y'),
```

Comment:
We should probably change `l j F Y` to `DateTimeInterface::ATOM` (or something else that looks like ISO 8601) whenever we are doing something with dates that humans are not directly involved with.

Reply:
Done in af35b1ea2eb2846bae0aa0da6fd79b8b83add97e